start

```diff
@@ -181,7 +263,45 @@ load(
     "rules_haskell_toolchains",
 )
 
+# Download alex and make it accessible as @alex.
+# Delete this block, alex.BUILD.bazel, and the reference in
+# stack_snapshot's tools if you don't use alex.
+http_archive(
+    name = "alex",
+    build_file = "//:$ALEX_BUILD_FILE",
+    sha256 = "d58e4d708b14ff332a8a8edad4fa8989cb6a9f518a7c6834e96281ac5f8ff232",
+    strip_prefix = "alex-3.2.4",
+    urls = ["http://hackage.haskell.org/package/alex-3.2.4/alex-3.2.4.tar.gz"],
+)
+
+load(
+    "@rules_haskell//haskell:cabal.bzl",
+    "stack_snapshot"
+)
+
+stack_snapshot(
+    name = "stackage",
+    extra_deps = {"zlib": ["@zlib.dev//:zlib"]},
+    packages = ["zlib"],
+    tools = [ "@alex" ],
+    snapshot = "lts-14.27", # Last snapshot published for ghc-8.6.5
+    # the default version picked up by rules_haskell
+)
+
 $(get_toolchain)
+
+$(get_zlib)
+EOF
+
+echo "Creating $ALEX_BUILD_FILE" >&2
+cat > "$ALEX_BUILD_FILE" <<EOF
+load("@rules_haskell//haskell:cabal.bzl", "haskell_cabal_binary")
+
+haskell_cabal_binary(
+    name = "alex",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
```

Comment:
```suggestion
haskell_cabal_binary(
    name = "alex",
    srcs = glob(["**"]),
    verbose = False,
    visibility = ["//visibility:public"],
)
```
It makes sense to set `verbose = False`, since it's an external dependency

Reply:
Done in 7c8135e023f585da4c17e05d17ad0664c2ce69e3